### PR TITLE
DOC-454 | Best practices for upload backups to cloud storage

### PR DESCRIPTION
**Pull request by nerpaula**

### CODE REVIEW

**Review by xyll — APPROVED** [review]
LGTM

File: arangograph/backups.md [hunk 1]
```diff
@@ -67,14 +67,38 @@ By default, a backup is not uploaded to the cloud, instead it remains on the
...
+#### Best practices for uploading backups
+
+When utilizing the **Upload backup to cloud storage** feature, a recommended
+approach is to implement a backup strategy that balances granularity and storage
+efficiency.
+
+One effective strategy involves creating a combination of backup intervals and
+retention periods. For instance, consider the following example:
+
+1. Perform 24 hourly backups with a retention period of 24 hours. Hourly backups
```
Comment (mattcarabine) [inline arangograph/backups.md:87]: Discussed with @xyll and think we'd prefer the following instead (please wordsmith to make better):

1. Backup every 4 hours, retained for a day
2. Backup every day, retained for a week
3. Backup every week, retained for a month
4. Backup every month, retained for a year

The original hourly backups will likely be too expensive right now, and not having weekly backups is a bit of a gap in the current propsal, so this fixes both issues.

**Review by mattcarabine — CHANGES_REQUESTED** [review]

**Comment by mattcarabine** [comment]
BTW we have raised https://arangodb.atlassian.net/browse/OAS-7617 to have these examples baked into the product to help people do this.

File: arangograph/backups.md (hunk 1 above)
Comment (nerpaula) [reply arangograph/backups.md:87]: Yes, makes sense. Will adjust it. And I think it's great to have these examples in the platform as well!

Comment (nerpaula) [reply arangograph/backups.md]: @mattcarabine can you please re-review?

File: arangograph/backups.md [hunk 2]
```diff
@@ -67,14 +67,40 @@ By default, a backup is not uploaded to the cloud, instead it remains on the
...
+   any point within the past week.
+3. Perform a backup every week with a retention period of a month. Weekly
+   backups allow you to recover from more extensive data.
+4. Perform a backup every month with a retention period of a year. Monthly 
+   backups provide a long-term perspective, enabling you to restore data from
+   any month within the past year.
+
+This backup strategy offers good granularity, providing multiple recovery
+options for different timeframes. By implementing this approach, you have a 
+total number of backups that is considerable lower in comparison to other
```
Comment (mattcarabine) [inline arangograph/backups.md:101]: Typo: considerable

**Review by mattcarabine — APPROVED** [review]

File: arangograph/backups.md (hunk 2 above)
Comment (nerpaula) [reply arangograph/backups.md:101]: LGTM?!

**Pull request merged**